packages/admin/src/targetGroups/addContacts/AddContactsGridSelect.tsx

```diff
@@ -0,0 +1,311 @@
+import { useMutation, useQuery } from "@apollo/client";
+import {
+    CancelButton,
+    Field,
+    FinalForm,
+    Toolbar,
+    ToolbarActions,
+    ToolbarFillSpace,
+    ToolbarItem,
+    ToolbarTitleItem,
+    useBufferedRowCount,
+    useDataGridRemote,
+    usePersistentColumnState,
+} from "@comet/admin";
+import { Add, Close, Remove, Save } from "@comet/admin-icons";
+import { ContentScopeInterface } from "@comet/cms-admin";
+import { Button, Dialog, DialogActions, DialogTitle, IconButton, useTheme } from "@mui/material";
+import { Box } from "@mui/system";
+import { DataGrid, GridColDef, GridToolbarQuickFilter } from "@mui/x-data-grid";
+import * as React from "react";
+import { FormattedMessage, IntlShape, useIntl } from "react-intl";
+import { MemoryRouter } from "react-router";
+
+import { targetGroupFormNamedOperations } from "../TargetGroupForm";
+import {
+    addBrevoContactsToTargetGroupMutation,
+    assignedBrevoContactsQuery,
+    removeBrevoContactFromTargetGroupMutation,
+} from "./AddContactsGridSelect.gql";
+import {
+    GQLAddBrevoContactsToTargetGroupMutation,
+    GQLAddBrevoContactsToTargetGroupMutationVariables,
+    GQLBrevoContactsGridQuery,
+    GQLBrevoContactsGridQueryVariables,
+    GQLRemoveBrevoContactFromTargetGroupMutation,
+    GQLRemoveBrevoContactFromTargetGroupMutationVariables,
+    GQLTargetGroupBrevoContactsListFragment,
+    namedOperations,
+} from "./AddContactsGridSelect.gql.generated";
+
+interface OpenSelectDialogToolbarProps {
```

Comment:
This should be called `AssignedContactsGridToolbar`

Reply:
I removed the interface, since I removed the intl property the interface did not make sense anymore to be used in just 1 function for 1 property. 
[794246a](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/794246a9b7e475bd91404a2b60cff933004a142e)